internal/apps/dop/providers/search/search_test.go

```diff
@@ -0,0 +1,535 @@
+// Copyright (c) 2021 Terminus, Inc.
```

Comment:
Bad test mock. The reviewers cannot find out what your are testing indeed.

Reply:
move common mock to `internal/pkg/mock`.

Reply:
adjusted